src/Client.php

```diff
@@ -183,24 +183,21 @@ public function getScheduleForDay(array $channelIds, Carbon $date)
             foreach ($data->items as $item) {
 
                 $scheduleItem = new ScheduleItem();
```

Comment:
You might wish to put the `$scheduleItem = new ScheduleItem();` line within the `if` too for consistency, though it should work as is.